### PR TITLE
Update available version of Mariadb to 11.3

### PR DESCRIPTION
**Pull request by uberhacker**
Also ran `npm audit fix` to remove vulnerability warnings.
```
5 vulnerabilities (1 moderate, 3 high, 1 critical)
```
Example output:
```
lando ssh -s defaults -c "mysql -V | grep 11.3.2"
mysql from 11.3.2-MariaDB, client 15.2 for Linux (x86_64) using readline 5.1
```

### CODE REVIEW

**Review by reynoldsalec — DISMISSED** [review]

**Comment by uberhacker** [comment]
Thank you for the quick review @reynoldsalec!  Should we check the boxes at https://github.com/lando/mariadb/issues/51?

**Comment by reynoldsalec** [comment]
@uberhacker this PR doesn't address #51; that issue is dedicated to updating the version of the `@lando/mariadb` used across all the recipes that consume it.

**Comment by uberhacker** [comment]
Oh, so I need to make some more changes to address [#51](https://github.com/lando/mariadb/issues/51)?

**Comment by reynoldsalec** [comment]
Yeah, you'd need to run those steps against the list of plugins on that issue to update their version of MariaDB. I can probably automate some of that if you want me to generate a bunch of PRs that you can go and review/edit, LMK!

- Acquia
- Backdrop
- Drupal
- Joomla
- Laravel
- Pantheon
- WordPress

**Comment by uberhacker** [comment]
Sure, I'm all for automation especially if it relates to version updates!

**Comment by uberhacker** [comment]
In the meantime, I converted this to draft until we can address all the issues in #51.

File: package.json [hunk 1]
```diff
@@ -42,11 +42,12 @@
     "test": "npm run lint && npm run test:unit"
   },
   "dependencies": {
+    "@lando/mariadb": "^1.2.0",
```
[the remark this answers is not in the document]
Comment (reynoldsalec) [reply package.json:45]: Good catch @rubenvarela; yeah that should be removed (and the package-lock regenerated). I think this was just part of the confusion around the steps in #51 and what the goal of the issue was, sorry @uberhacker about the mix-up there.

[the remark this answers is not in the document]
Comment (uberhacker) [reply package.json]: This should be resolved.  Thanks for the feedback @rubenvarela!

**Review by reynoldsalec — DISMISSED** [review]

**Review by reynoldsalec — APPROVED** [review]

**Pull request merged**